config/process_group.go

```diff
@@ -4,7 +4,7 @@ import (
 	"bytes"
 	"strings"
 
-	"github.com/ochinchina/supervisord/util"
+	"github.com/cyralinc/supervisord/util"
```

Comment:
Again, I'd avoid this change and use a replace directive in go.mod instead - the goal being to make minimal changes to .go files so that merging upstream changes is easy.